components/frontend/src/header_footer/buttons/DatePickerButton.js

```diff
@@ -0,0 +1,48 @@
+import EventIcon from "@mui/icons-material/Event"
+import { Button, Menu, Tooltip } from "@mui/material"
+import { StaticDatePicker } from "@mui/x-date-pickers/StaticDatePicker"
+import { func } from "prop-types"
+import { useState } from "react"
+
+import { datePropType } from "../../sharedPropTypes"
+
+export function DatePickerButton({ onChange, reportDate }) {
+    const [anchorEl, setAnchorEl] = useState()
+    const handleMenu = (event) => setAnchorEl(event.currentTarget)
+    const handleClose = () => setAnchorEl(null)
+    return (
+        <Tooltip placement="left" title="Show the report as it was on the selected date">
+            <span /* https://mui.com/material-ui/react-tooltip/#disabled-elements */>
```

Comment:
This comment seems unneeded here, or does it refer to `disableFuture`?

Reply:
No, it explains why there is an otherwise useless span.